setup.py

```diff
@@ -242,7 +242,7 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
     'grpcio>=1.15.0',
 ]
 hdfs = [
-    'snakebite>=2.7.8',
+    'snakebite-py3>=3.0.5',
```

Comment:
Seems fine for master -- do you know if this py3 module also supports py2, or do we need to have the `;python_version>="3"` trick on 1.10.8 to support py2 and 3?

Reply:
It would be safer to use "if" in 1.10.8. I will do it when cherry-picking.